### ISSUE TITLE
debops.mariadb : Create databases failed

### ISSUE THREAD

**Issue by zedrickvillas** [carlalexander/debops-wordpress#110]
I am not on the 3rd step of the installation but encountered this problem. I saw a closed issue regarding this but was not able to understand the direct fix for this.

> [debops.mariadb : Create databases] ***************************************
fatal: [thehealthhacks.com]: UNREACHABLE! => {"changed": false, "msg": "All items completed", "results": [{"_ansible_item_result": true, "item": {"name": "wordpress"}, "msg": "Failed to connect to the host via ssh: ssh: Could not resolve hostname thehealthhacks: nodename nor servname provided, or not known\r\n", "unreachable": true}]}
failed: [thehealthhacks.com] (item={u'name': u'wordpress'}) => {"item": {"name": "wordpress"}, "msg": "Failed to connect to the host via ssh: ssh: Could not resolve hostname thehealthhacks: nodename nor servname provided, or not known\r\n", "unreachable": true}

The ansible was successfully connected via ssh but this tasks seems to be disconnected and i don't know why.

**Comment by carlalexander** [comment]
Are you talking about this [issue](https://github.com/carlalexander/debops-wordpress/issues/76)? Could you show me your configuration? I'm curious because the error says "Could not resolve hostname thehealthhacks" and not `thehealthhacks.com`.

**Comment by zedrickvillas** [comment]
> Are you talking about this issue?
> Yes, but cant seem to understand the link between my problem and theirs.

What config? The inventory file?

**Comment by carlalexander** [comment]
Yes! And you're not using any other configuration files?

**Comment by zedrickvillas** [comment]
Here is my inventory file:

> [debops_all_hosts]
> thehealthhacks.com
> [wordpress]
> thehealthhacks.com

Yes. I didn't modify other configuration file other than the fix for plug_src and the inventory file.

**Comment by zedrickvillas** [comment]
tried to start from scratch but still blocked by the error.
this time i used the host www.thehealthhacks.com and now the error is 

> TASK [debops.mariadb : Create databases] ***************************************
> failed: [www.thehealthhacks.com](item={u'name': u'wordpress'}) => {"item": {"name": "wordpress"}, "msg": "Failed to connect to the host via ssh: ssh: Could not resolve hostname www: nodename nor servname provided, or not known\r\n", "unreachable": true}
> fatal: [www.thehealthhacks.com]: UNREACHABLE! => {"changed": false, "msg": "All items completed", "results": [{"_ansible_item_result": true, "item": {"name": "wordpress"}, "msg": "Failed to connect to the host via ssh: ssh: Could not resolve hostname www: nodename nor servname provided, or not known\r\n", "unreachable": true}]}

**Comment by zedrickvillas** [comment]
I was now able to fix my earlier problem by hardcoding the host name into mariadb config on ymls.
but i've encountered another error >_<

will create another issue of what i've encountered.

**Comment by zedrickvillas** [comment]
Closing this issue.

Note:
This issue only occurs on LiNode Servers.

Recommendation:
Use DigitalOcean :D
